org/ystia/yorc/yorc/pub/types.yaml

```diff
@@ -169,6 +169,28 @@ node_types:
       config:
         type: org.ystia.yorc.pub.capabilities.YorcConfigContainer
 
+  org.ystia.yorc.pub.TerraformConfig:
+    derived_from: tosca.nodes.Root
+    description: Terraform configuration for Yorc
+    abstract: true
+    properties:
+      plugins_dir:
+        description: "The directory where are located Terraform plugins"
+        type: string
+        required: true
+        default: /var/terraform/plugins
+      plugins_download_urls:
```

Comment:
I think we should add the downloads urls of plugins that are required by yorc as the default value of this property (as it is done in the topologies)

Reply:
ok